concepticondata/concepticon.tsv

```diff
@@ -1082,18 +1082,18 @@ ID	GLOSS	SEMANTICFIELD	DEFINITION	ONTOLOGICAL_CATEGORY	REPLACEMENT_ID
 1081	BILL	Possession	Sum owed in a restaurant, or the relative bill.	Person/Thing	
 1082	MERCHANT	Possession	A businessperson engaged in trade.	Person/Thing	
 1083	MESS	The physical world	A state of confusion and disorderliness.	Person/Thing	
-1084	CALL	Speech and language	To order or summon by using one's voice.	Action/Process	
+1084	CALL	Speech and language	To ask or order someone to come.	Action/Process	
```

Comment:
This is a difficult concept to define, but from what I remember from the datasets I've worked on the original meaning seems closer to what conceptlists tend to imply. In particular, I'd at least at the hierarchical relationship expressed by original, at least as an alternative, with something like "To ask or summon someone"; at the same time, a "CALL" is not necessarily used to motivate a movement, but can be more generally to ask for attention and thus establish a communication.

Reply:
You are right. I think I'll change it to "To address or summon someone". But I'll open an issue for that concept anyway since some of the linked glosses might have even a more different meaning (closer to "scream").

Reply:
Great. We knew this would necessarily imply some remapping in the conceptlists, anyway. :+1: